e2e/test/scenarios/embedding-sdk/static-dashboard-cors.cy.spec.js

```diff
@@ -0,0 +1,199 @@
+import { USERS } from "e2e/support/cypress_data";
```

Comment:
copy the old `static-dashboard.cy.spec.js` from the last commit to this new file, nothing has changed except calling the new story that uses the new decorator that won't sign the JWT on storybook